hw_diag/views/diagnostics.py

```diff
@@ -54,21 +54,13 @@ def get_diagnostics():
     )
 
 
-def is_gwmfr_running():
-    return not os.path.isfile(ECC_SUCCESSFUL_TOUCH_FILEPATH)
```

Comment:
`ECC_SUCCESSFUL_TOUCH_FILEPATH` declaration can also be removed https://github.com/NebraLtd/hm-diag/pull/171/files#diff-04b1aca2fd0e3d6dca6921bc407e7e1cf9b6c93e823d1d31591fcfd6b398304aL22